src/exercises/day-3/safe-ffi-wrapper.rs

```diff
@@ -142,7 +141,6 @@ mod tests {
     use std::error::Error;
 
     #[test]
-    #[cfg(not(miri))]
```

Comment:
These might be nice to have, else `cargo +nightly miri test` fails (at least on MacOS)